From 4499e109ecfbace80362aab185095bdfb1038ecf Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Fri, 31 Aug 2018 05:22:29 +0200
Subject: [PATCH 072/293] xlat: update resources

* xlat/resources.in: Declare as #value indexed, provide fallback values.
(RLIMIT_VMEM): Remove.
---
 xlat/resources.in | 56 ++++++++++++++++++++++++++++++++++++++-----------------
 1 file changed, 39 insertions(+), 17 deletions(-)

diff --git a/xlat/resources.in b/xlat/resources.in
index 8c45b7f..f12cb85 100644
--- a/xlat/resources.in
+++ b/xlat/resources.in
@@ -1,17 +1,39 @@
-RLIMIT_AS
-RLIMIT_CORE
-RLIMIT_CPU
-RLIMIT_DATA
-RLIMIT_FSIZE
-RLIMIT_LOCKS
-RLIMIT_MEMLOCK
-RLIMIT_MSGQUEUE
-RLIMIT_NICE
-RLIMIT_NOFILE
-RLIMIT_NPROC
-RLIMIT_RSS
-RLIMIT_RTPRIO
-RLIMIT_RTTIME
-RLIMIT_SIGPENDING
-RLIMIT_STACK
-RLIMIT_VMEM
+#value_indexed
+RLIMIT_CPU		0
+RLIMIT_FSIZE		1
+RLIMIT_DATA		2
+RLIMIT_STACK		3
+RLIMIT_CORE		4
+
+#if defined(__alpha__)
+RLIMIT_RSS		5
+RLIMIT_NOFILE		6
+RLIMIT_AS		7
+RLIMIT_NPROC		8
+RLIMIT_MEMLOCK		9
+#elif defined(__mips__)
+RLIMIT_NOFILE		5
+RLIMIT_AS		6
+RLIMIT_RSS		7
+RLIMIT_NPROC		8
+RLIMIT_MEMLOCK		9
+#elif defined(__sparc__)
+RLIMIT_RSS		5
+RLIMIT_NOFILE		6
+RLIMIT_NPROC		7
+RLIMIT_MEMLOCK		8
+RLIMIT_AS		9
+#else
+RLIMIT_RSS		5
+RLIMIT_NPROC		6
+RLIMIT_NOFILE		7
+RLIMIT_MEMLOCK		8
+RLIMIT_AS		9
+#endif
+
+RLIMIT_LOCKS		10
+RLIMIT_SIGPENDING	11
+RLIMIT_MSGQUEUE		12
+RLIMIT_NICE		13
+RLIMIT_RTPRIO		14
+RLIMIT_RTTIME		15
-- 
1.7.12.4

